bare/roles/web/tasks/mastodon.yml

```diff
@@ -0,0 +1,97 @@
+---
+- name: Clone mastodon
+  git:
+    repo: "https://github.com/tootsuite/mastodon.git"
+    dest: "~/live"
```

Comment:
This should be configurable. It's the most likely candidate for a value people would want to customize.